packages/components/odf-resources/ButtonBar.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```

Comment:
every file that is under `components/odf-resources` need to go into `shared` directory...

Reply:
see example of how it is done in `odf-console` repo...

Reply:
moved to respective directory as mentioned in ODF console.